salalib/axialmap.cpp

```diff
@@ -261,11 +259,10 @@ bool ShapeGraph::integrate(Communicator *comm, const pvecint& radius_list, bool
    }
 
    // first enter the required attribute columns:
-   size_t r;
-   for (r = 0; r < radius.size(); r++) {
+   for (int radius: radii) {
       std::string radius_text;
-      if (radius[r] != -1) {
-         radius_text = std::string(" R") + dXstring::formatString(int(radius[r]),"%d");
+      if (radius != -1) {
+         radius_text = std::string(" R") + dXstring::formatString(int(radius),"%d");

```

Comment:
Wouldn't that be more efficiently written as `dXstring::formatString(radius, " R%d")`?
Also, the cast from `int` to `int` is pointless.